### ISSUE TITLE
Gs at the beginning of the reads

### ISSUE THREAD

**Issue by fgypas** [timbitz/aligater#8]
Hello again. I have a new question and this time is related to trimming of the reads. It was mentioned to me that due to the preparation (i think) of the samples, some Gs are introduced at the beginning of the reads. Indeed when I checked it there is a big portion of reads like this. The question is, should I trim a fixed number of Gs or everything that starts with G should be trimmed?

**Comment by timbitz** [comment]
With our LIGR-Seq data the first thing I do is filter for unique reads including the random barcode, then I trim 5nt off the front and run the rest of the pipeline after that.

**Comment by timbitz** [comment]
I don't think it will cause a huge problem though if you leave them in.  The alignment steps should be lenient enough to allow non-aligning segments on the 5' and 3' ends of reads.

**Comment by fgypas** [comment]
5nt when you have Gs or in general?

**Comment by timbitz** [comment]
Our raw data has an 5' random barcode that I trim in general, is it possible that your library prep is different from ours? Either way I don't think it will affect how `aligater` runs, it is designed to allow soft clipping on the ends.
